### ISSUE TITLE
[Suggestion] Accordants arms

### ISSUE THREAD

**Issue by Oldarorn** [PainedPsyche/cui_character#1]
Hello, I saw in the README you had some problems with gloves ! I remember somebody did a modification with skinchanger where gloves were accorded to each t_shirt and torsos ! I thought it could help you :)

here's a link : https://github.com/Sadler2/skinchanger/blob/master/client/accordant_arms.lua

Your release looks amazing, you did a great job thank you

**Comment by PainedPsyche** [comment]
That's very interesting and I think it will save me a lot of work when trying to implement gloves. Thanks for posting this.

**Comment by MadLad1Clip** [comment]
Yeah would love to see the gloves added to this system. This has to be one of the cleanest systems i have found to create characters. Keep up the good work.

**Comment by DonJoan** [comment]
See pull requests I add gloves

**Comment by DonJoan** [comment]
And I will soon add backpacks
